### DISCUSSION TITLE
How to clear the framework cache?

### DISCUSSION THREAD

**Discussion by tobmo** [firefly-iii/firefly-iii#7721] [Firefly III] [answered]
Hi,

while using firefly-iii the storage/framework/cache/data folder grew to several tens of GB. How do I delete the cache? Simply deleting the folder results in a broken installation.

Thank you for the help.

**Comment by JC5** [comment, marked as answer]
Browse to `/flush`, that should do it. Emptying the folder should not result in errors; what error do you get?

**Reply by tobmo** [reply]
Thank you, `/flush` seems to be doing something. However, it stops when it exceeds the maximum 30s execution time. I suppose I just have to run it multiple times?

Simply deleting the cache in the filesystem results in errors like this:
`file_put_contents(/var/www/firefly/storage/framework/cache/data/79/42/7942698bef726131c5478e8a9c79d865c6c9ad9a): failed to open stream: No such file or directory`

**Reply by JC5** [reply]
Try emptying the folder, and then browse to /flush. An alternative is to use `php artisan cache:clear` on the command line.

**Reply by tobmo** [reply]
I ran /flush a couple of times. The cache is fully cleared now. Thank you.
